server/views/components/need-score-indicator/template.njk

```diff
@@ -1,5 +1,5 @@
 <div class="needs-score-label__card">
-    <p><span class="govuk-!-font-weight-bold">{{ needsScore.score }}</span>
-    out of {{ needsScore.upperBound }}</p>
+    <p><span class="govuk-!-font-weight-bold govuk-heading-s needsScoreMain">{{ needsScore.score }}</span>
```

Comment:
because you're adding your own class here you can remove the other two classes and change your CSS so it looks like this:
```css
.needsScoreMain {
  @extend .govuk-\!-font-weight-bold;
  @extend .govuk-heading-s;
  margin-bottom: 0;
}
```